src/tests/ftest/server/metadata.py

```diff
@@ -306,78 +424,65 @@ def test_metadata_server_restart(self):
                 # Start the servers
                 self.start_server_managers()
 
-    @avocado.fail_on(DaosApiError)
+        self.log.info("Test passed")
+
     def test_container_removal_after_der_nospace(self):
-        """JIRA ID: DAOS-4858
+        """JIRA ID: DAOS-4858.
 
         Test Description:
            Verify container can be successfully deleted when the storage pool
```

Comment:
This test seems a little strange, does not seem to be doing what is specified in https://daosio.atlassian.net/browse/DAOS-4858 

Test as written seems to exercise metadata capacity in overlapping ways to metadata_fillup and addremove tests. Also the ticket seems to describe exercising regular I/Os on the pool target storage capacity. Versus what is implemented so far is exercising metadata capacity only.

A note for the future, not an issue for this PR.

Reply:
Created https://daosio.atlassian.net/browse/DAOS-8484 to track this.